packages/vtree/core/modeling/operations/transforms/index.js

```diff
@@ -0,0 +1,28 @@
+module.exports = {
+  center: require('./center').center,
+  centerX: require('./center').centerX,
```

Comment:
Are these aliases necessary?
What happens if there’s only center(), etc?

Reply:
not sure I follow ?
I wish we could only provide the basic, but VTREE acts as API, so anything is missing in API is not useable for users :(